azure/cosmosdb/remote.tf

```diff
@@ -5,3 +5,13 @@ data "terraform_remote_state" "network" {
     path = "../network/terraform.tfstate"
   }
 }
+
+data "terraform_remote_state" "kubernetes" {
```

Comment:
Just a question. cosmosdb depends on kubernetes?
I thought we create in the following order.
network -> cosmosdb -> kubernetes

Reply:
>cosmosdb depends on kubernetes?

Yes, Because we need to allow access to cosmosdb from k8s node subnet, but this k8s subnet is created by kubernetes module.

There are two ways to create it.
- network -> cosmosdb (default) -> kubernetes (use_cosmosdb = true) -> cosmosdb (use_kubernetes = true)
- network -> kubernetes (use_cosmosdb = true) => cosmosdb (use_kubernetes = true)